### PR TITLE
feat(eventState): add default value `sequential` for OnEvents.ActionMode 

### PR DESCRIPTION
**Pull request by lsytj0413**
Signed-off-by: lsytj0413 <511121939@qq.com>

**Many thanks for submitting your Pull Request :heart:!**

**What this PR does / why we need it**:

- add OnEvents default ActionMode to `sequential`

**Special notes for reviewers**:

**Additional information (if needed):**

### CODE REVIEW

File: model/workflow.go [hunk 1]
```diff
@@ -19,13 +19,21 @@ import (
...
-	// ActionModeSequential ...
+	// ActionModeSequential specifies actions should be performed in sequence
 	ActionModeSequential ActionMode = "sequential"
-	// ActionModeParallel ...
+
+	// ActionModeParallel specifies actions should be performed in parallel
 	ActionModeParallel ActionMode = "parallel"
+)
+
+const (
```
Comment (spolti) [inline model/workflow.go:33]: is there a reason to break it into two `const` blocks?

**Review by spolti — COMMENTED** [review]
Besides splinting the event_state into its own go file, what do you mean by "add defaults mode"?

**Comment by ricardozanini** [comment]
@lsytj0413 can you please rebase?

File: model/workflow.go (hunk 1 above)
Comment (lsytj0413) [reply model/workflow.go:44]: same as https://github.com/serverlessworkflow/sdk-go/pull/97#discussion_r987872135

**Comment by lsytj0413** [comment]
> Besides splinting the event_state into its own go file, what do you mean by "add defaults mode"?

Correct the message to `add default value sequential for OnEvents.ActionMode`

**Comment by lsytj0413** [comment]
> @lsytj0413 can you please rebase?

done, PTAL

**Review by ricardozanini — APPROVED** [review]

**Pull request merged**